src/placeos-core/module_manager.cr

```diff
@@ -275,37 +275,54 @@ module PlaceOS::Core
     # Clustering
     ###############################################################################################
 
-    def on_managed_edge?(mod : Model::Module)
-      mod.on_edge? && own_node?(mod.edge_id.as(String))
-    end
-
-    private getter semaphore = Atomic(Int32).new(0)
+    private getter queued_stabilization_events = Atomic(Int32).new(0)
     private getter stabilize_lock = Mutex.new
+    private STABILIZE_BATCH_SIZE = 32
```

Comment:
Any reason for this batch size?

Reply:
None in particular. Could go higher to 64, just thought it was a good middle ground.
What do you reckon?

Reply:
NFI. Was mainly curious. May be worth dropping an `# OPTIMIZE:` note in there to test some different options down the track if the best is currently unknown.